### ISSUE TITLE
Dry run option

### ISSUE THREAD

**Issue by fharper** [manusa/mnIMAPSync#34]
Before using a tool like this one, that could potentially mess up email accounts (no disrespect about your tool, just that it's a reality, and it's a lot of trust the users give you), it would be nice to be able to do a dry run. In other words, running the app, mimicking the operations without having real effects.

**Comment by manusa** [comment]
Yes, this is would be a nice feature to have, although I'm not sure what the expected output would be.

The application basically indexes messages from both accounts and then performs a sync for diffs.

Maybe a dump of the list of messages to be copied + deleted could be useful.

**Comment by fharper** [comment]
It's what other tools I've used before were doing.
